jerry-libc/jerry-libc.c

```diff
@@ -27,7 +27,7 @@
 /**
  * State of pseudo-random number generator
  */
-static uint32_t libc_random_gen_state[4] = { 1455997910, 1999515274, 1234451287, 1949149569 };
+ static uint32_t libc_random_gen_state[4] = { 1455997910, 1999515274, 1234451287, 1949149569 };
```

Comment:
This leading space seems to be a leftover / addition by mistake. Probably Travis does not like it either.